pkg/stanza/operator/input/windows/input.go

```diff
@@ -283,42 +262,37 @@ func (i *Input) processEvent(ctx context.Context, event Event) {
 		return
 	}
 
-	formattedEvent, err := event.RenderFormatted(i.buffer, publisher)
+	deepEvent, err := event.RenderDeep(i.buffer, publisher)
 	if err != nil {
 		i.Logger().Error("Failed to render formatted event", zap.Error(err))
 		i.sendEvent(ctx, simpleEvent)
 		return
 	}
-	formattedEvent.RemoteServer = remoteServer
-	i.sendEvent(ctx, formattedEvent)
+	i.sendEvent(ctx, deepEvent)
 }
 
 // sendEvent will send EventXML as an entry to the operator's output.
-func (i *Input) sendEvent(ctx context.Context, eventXML EventXML) {
-	body := eventXML.parseBody()
-	entry, err := i.NewEntry(body)
+func (i *Input) sendEvent(ctx context.Context, eventXML *EventXML) {
+	// body := eventXML.parseBody()
+	var body any = eventXML.Original
+	if !i.raw {
+		body = formattedBody(eventXML)
+	}
+
+	e, err := i.NewEntry(body)
 	if err != nil {
 		i.Logger().Error("Failed to create entry", zap.Error(err))
 		return
 	}
 
-	entry.Timestamp = eventXML.parseTimestamp()
-	entry.Severity = eventXML.parseRenderedSeverity()
-	_ = i.Write(ctx, entry)
-}
+	e.Timestamp = parseTimestamp(eventXML.TimeCreated.SystemTime)
+	e.Severity = parseSeverity(eventXML.RenderedLevel, eventXML.Level)
 
-// sendEventRaw will send EventRaw as an entry to the operator's output.
-func (i *Input) sendEventRaw(ctx context.Context, eventRaw EventRaw) {
-	body := eventRaw.parseBody()
-	entry, err := i.NewEntry(body)
-	if err != nil {
-		i.Logger().Error("Failed to create entry", zap.Error(err))
-		return
+	if i.remote.Server != "" {
+		e.Attributes["remote_server"] = i.remote.Server
```

Comment:
I think the [`server.address` attribute](https://github.com/open-telemetry/semantic-conventions/blob/881b3f0b80b3845a8cf367077554c6a1f1be5c7d/docs/attributes-registry/server.md) may be appropriate here. Any opinions on this?

Reply:
It seems reasonable to me.